atom/browser/api/atom_api_app.h

```diff
@@ -178,8 +184,12 @@ class App : public AtomBrowserClient::Delegate,
 
   void SetDesktopName(const std::string& desktop_name);
   std::string GetLocale();
-  bool MakeSingleInstance(
-      const ProcessSingleton::NotificationCallback& callback);
+  bool OnSecondInstance(const base::CommandLine::StringVector& cmd,
+                        const base::FilePath& cwd);
+  bool IsPrimaryInstance(mate::Arguments* args);
+  bool IsSingleInstance();
```

Comment:
If we _do_ go with this three-method API ... these two functions have names which are confusingly similar to each other. Since we're trying to make this readable without reaching for the docs, is there a clearer way to name these?